vmdb/app/models/customization_script.rb

```diff
@@ -1,4 +1,7 @@
 class CustomizationScript < ActiveRecord::Base
   include NewWithTypeStiMixin
   belongs_to :provisioning_manager
+
+  scope :ptables, where(:type => "CustomizationScriptPtable")
+  scope :media, where(:type => "CustomizationScriptMedium")
```

Comment:
I think it's preferable to just use normal class methods instead of scope.  i.e. A class method that returns an AR::Relation is a scope.